lib/settings/pages/about_settings_page.dart

```diff
@@ -45,10 +46,10 @@ class AboutSettingsPage extends StatelessWidget {
                     'GitHub',
                     style: theme.textTheme.titleMedium?.copyWith(fontWeight: FontWeight.w600),
                   ),
-                  subtitle: const Text('github.com/hjiangsu/thunder'),
+                  subtitle: const Text('github.com/thunder-app/thunder'),
```

Comment:
@hjiangsu If you intentionally wanted to keep a reference to your name/username in the app (which you deserve), I would put something like "developed by hjiangsu" beneath the app title on the about page, rather than using the old GitHub link.

Reply:
I think we can change it to the org name, it makes more sense to do that.

If anything I was thinking we could just have a separate section where we can attribute main contributors of the project (at this point, this would include you, @ajsosa, @CTalvio, @machinaeZER0  and maybe others that have helped with this project!)